ExtIO_sddc/resource.h

```diff
@@ -77,7 +77,7 @@
 #define IDC_STATIC                      -1
 
 
-#ifndef _DEBUG  // define in processor setup resource !
+#ifdef NDEBUG  // define in processor setup resource !
```

Comment:
maybe we define this out of resource.h.

I can see _DEBUG is defined. which may not apply to resource compile.
<PreprocessorDefinitions>TRACE;WIN32;_DEBUG;EXTIORX103_EXPORTS;_WINDOWS;_USRDLL;_CRT_SECURE_NO_WARNINGS;%(PreprocessorDefinitions)</PreprocessorDefinitions>

Reply:
here my one 
TRACE;WIN32;_DEBUG;_WINDOWS;_USRDLL;_CRT_SECURE_NO_WARNINGS;%(PreprocessorDefinitions)
In italy seems not to work ' may be the "language" :-)) 
EXTIORX103_EXPORTS I think can be removed